sci-libs/camd/camd-2.4.6.ebuild

```diff
@@ -4,7 +4,7 @@
 EAPI=7
 
 DESCRIPTION="Library to order a sparse matrix prior to Cholesky factorization"
-HOMEPAGE="http://faculty.cse.tamu.edu/davis/suitesparse.html"
+HOMEPAGE="https://faculty.cse.tamu.edu/davis/suitesparse.html"
```

Comment:
Site does not provide a secure connection... are you checking these websites vice just adding https?